doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -213,27 +219,26 @@ There is dedicated syntax for list and array literals.
 Ltac Definitions
 ~~~~~~~~~~~~~~~~
 
-.. cmd:: Ltac2 {? mutable} {? rec} @lident := @ltac2_value
+.. cmd:: Ltac2 {? mutable } {? rec } @tac2def_body {* with @tac2def_body }
    :name: Ltac2
 
+   .. insertprodn tac2def_body ltac2_input_fun
+
+   .. prodn::
+      tac2def_body ::= {| _ | @ident } {* @ltac2_input_fun } := @tac2expr6
+      ltac2_input_fun ::= @tac2pat0
+
    This command defines a new global Ltac2 value.
```

Comment:
Since it is now documented in the syntax, the usual possibility to use syntactic sugar to put arguments of a function before the `:=` (and thus to omit the `fun` keyword) should be mentioned in the text.